orchestra/tests/test_dashboard.py

```diff
@@ -574,16 +567,3 @@ def _verify_good_task_assignment_information(
             }
         }
         self.assertEquals(returned, expected)
-
-    def _submit_assignment(self, client, task_id, data=None,
```

Comment:
moved this into the base OrchestraTestCase class so that it could be used in latency reporting